src/shogun/regression/LeastAngleRegression.h

```diff
@@ -184,20 +188,49 @@ class CLeastAngleRegression: public CLinearMachine
 	virtual const char* get_name() const { return "LeastAngleRegression"; }
 
 protected:
-	virtual bool train_machine(CFeatures* data=NULL);
+	/**
+	* An interface method to - this is called by the superclass's (CLinearMachine)
```

Comment:
No need to explicitly mention the base class name here (i.e. remove CLinearMachine)